expan/core/experiment.py

```diff
@@ -339,3 +374,14 @@ def run_goodness_of_fit_test(self, observed_freqs, expected_freqs, alpha=0.01, m
             raise ValueError("Variant split check was cancelled since observed or expected frequencies "
                              "are less than 2.")
         return split_is_unbiased, p_value
+
+def _choose_threshold_type(data):
```

Comment:
preconditions, mostly for documentation? 
```assert data, "data should be defined"```

Reply:
good point, thank you, see below.

Reply:
`data = np.array([])` will throw on `data.min()`

Reply:
... aka developers playing compiler.

Reply:
can I resolve this?

Reply:
I'll resolve. The conversion to `pandas.Series` helps